arm-network/2016-09-01/swagger/network.json

```diff
@@ -1672,6 +1672,9 @@
             "description": "Parameters supplied to the create/update NetworkInterface operation"
           },
           {
+            "$ref": "#/parameters/LocationParameter"
```

Comment:
@ureesoriano - You are still not getting it. `location` is a top level property of the request payload whereas you have specified it inside properties. That is incorrect.

If you look at the example request payload in the msdn link that you have pointed then location is a top level property.
```
{
   "location": "North US", <<<<<<<<<<<<<<<<<<<<<<<<< location is here
   "tags": { "key": "value" },
   "properties": {
      "publicIPAllocationMethod": "Static",
      "publicIPAddressVersion": "IPv4",
      "idleTimeoutInMinutes": 4,
      "dnsSettings": {
         "domainNameLabel": "mylabel",
         "reverseFqdn": "contoso.com."
      }
   }
}
```

Again as per the REST principles a request cannot have two body parameters.

I